modules/fedimint-wallet-server/src/lib.rs

```diff
@@ -1246,7 +1255,7 @@ impl Wallet {
                     if Duration::from_secs(10) < duration {
                         warn!(target: LOG_MODULE_WALLET, duration_secs=duration.as_secs(), "Updating from bitcoind slow");
                     }
-                    let remaining = desired_frequency.saturating_sub(duration);
+                    let remaining = desired_poll_interval.saturating_sub(duration);
```

Comment:
[`tokio::time::Interval`](https://docs.rs/tokio/latest/tokio/time/struct.Interval.html) may be a good abstraction too here.

Reply:
Oh damn. I need to start using it. I need this exact thing all the time. Thanks @elsirion